src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -140,6 +140,40 @@
       (when (:show-nih-link-warning? props)
         [nih-link-warning/NihLinkWarning])])})
 
+;; GAWB-666 Globally show Queued and Cromwell (active) counts
+(defn queue-status-counts [workflow-counts]
+  (let [count-map (get workflow-counts "workflowCountsByStatus" (empty {}))
+        queue-time (get workflow-counts "estimatedQueueTimeMS" 0)]
+    {:queue-time queue-time
+     :queued (+ (get count-map "Queued" 0) (get count-map "Launching" 0))
+     :active (+ (get count-map "Submitted" 0) (get count-map "Running" 0) (get count-map "Aborting" 0))
+     :terminal (+ (get count-map "Failed" 0) (get count-map "Succeeded" 0) (get count-map "Aborted" 0))}))
+
+(react/defc GlobalSubmissionStatus
+  {:render
+   (fn [{:keys [state]}]
+     (when-let [status-counts (:status-counts @state)]
+       [:span {:style {:marginRight "1ex"}}
+         [:br {}],
+         "Submissions: ", (:queued status-counts), " Queued; ", (:active status-counts), " Active"]))
```

Comment:
This seems like strangely structured HTML.  I always get suspicious of \<br\> tags.  Make this span a div, and make the \<a\> (for the profile) not float.  To get the right justification, put `:textAlign "right"` on the div containing the \<a\> and [GlobalSubmissionStatus].  You can also move the `:marginRight "1ex"` to that div and get rid of it from the two children.

Also the commas are unnecessary.